packages/cli/src/commands/task.ts

```diff
@@ -34,7 +34,7 @@ import {
 } from '../utils/env-variables.js';
 
 const { prompt } = enquirer;
-const AGENT_IMAGE = 'ghcr.io/endorhq/rover/node:v1.1.0';
+const AGENT_IMAGE = 'ghcr.io/endorhq/rover/node:v1.3.0';
```

Comment:
This is the only thing that I'm not sure about... When this happens we should probably build manually once and push so that the image is available.

In this case is not critical (we could keep using v1.1.0 and bump afterwards, ) but still this opens the question on when to bump and when to build and push the images?

@Angelmmiguel

Reply:
As discussed I will create issues to reflect what we need to do for future releases :)